src/features/locators/components/LocatorIcon.tsx

```diff
@@ -22,7 +22,7 @@ export const LocatorIcon: React.FC<Props> = ({ message, locator, deleted }) => {
   const deletedIcon = <Trash size={14} color="#9a9da9" className="jdn__locatorsList-status" />;
 
   const failureIcon = (
-    <Tooltip title={locator.errorMessage ?? "Locator generation was failed"}>
+    <Tooltip title={message ?? "Locator generation was failed"}>
```

Comment:
I am not sure about this change.
`message` property is about validation after editing or creating, but `errorMessage` comes during locator generation

Reply:
thank you, fixed